src/transaction/utils.ts

```diff
@@ -200,6 +201,16 @@ function prepareTransaction(
     }
   }
 
+  function convertIssuedCurrencyToAccountIfPresent(fieldName: string): void {
+    const amount = txJSON[fieldName]
+    if (typeof amount  === 'number'
+     || amount instanceof Array
+     || amount === undefined)
+      return
```

Comment:
you have a potential bug here in that `amount` can be `null` in which case you will pass it to `toRippledAmount` which will likely raise. you can fix this by swapping the `amount === undefined` predicate to `amount == null` here. for this reason, i prefer to _always_ check for null/undefined using `== null` and never reference a literal undefined.